src/Nox.Generator/Application/IntegrationEvents/CustomIntegrationEvent.template.cs

```diff
@@ -12,13 +12,24 @@
 {{ attributes = integrationEvent.ObjectTypeOptions.Attributes -}}
 {{- end -}}
 // Generated
+{{func pascalCaseToCamelCase(pascal)
```

Comment:
To not have the same code in multiple places, we can maybe in the future try to export the functions in a separate file and import them if possible in the template we need to use.

Reply:
yes, will look into this in the future